### PR TITLE
COS-2705: block edges for versions with 4.18.0-372.88.1.el8_6 or later kernel

### PR DESCRIPTION
**Pull request by PratikMahajan**
After rebooting into kernel-4.18.0-372.88.1.el8_6 or later kernel nodes experience high load average and io_wait times 
Nodes may fail to start or stop pods, probes may fail 
Workload and host processes may become unresponsive and workload may be disrupted

### CODE REVIEW

File: blocked-edges/4.11.58-OldKernelHighLoadIOWait.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+to: 4.11.58
+from: .*
+
```
Comment (LalatenduMohanty) [inline blocked-edges/4.11.58-OldKernelHighLoadIOWait.yaml:3]: We do not need the extra new line here.

Comment (LalatenduMohanty) [inline blocked-edges/4.11.58-OldKernelHighLoadIOWait.yaml:3]: We do not need the extra new line here.

File: blocked-edges/4.11.58-OldKernelHighLoadIOWait.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+url: https://issues.redhat.com/browse/COS-2705
+name: OldKernelHighLoadIOWait
+message: "After rebooting into kernel-4.18.0-372.88.1.el8_6 or later kernel nodes experience high load average and io_wait times. The nodes might fail to start or stop pods and probes may fail. Workload and host processes may become unresponsive and workload may be disrupted"
```
Comment (wking) [inline blocked-edges/4.11.58-OldKernelHighLoadIOWait.yaml:6]:
```suggestion
message: "After rebooting into kernel-4.18.0-372.88.1.el8_6 or later, kernel nodes experience high load average and io_wait times. The nodes might fail to start or stop pods and probes may fail. Workload and host processes may become unresponsive and workload may be disrupted."
```

File: blocked-edges/4.11.58-OldKernelHighLoadIOWait.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+to: 4.11.58
+from: .*
+
+url: https://issues.redhat.com/browse/COS-2705
+name: OldKernelHighLoadIOWait
```
Comment (LalatenduMohanty) [inline blocked-edges/4.11.58-OldKernelHighLoadIOWait.yaml:5]: Suggestion `RHELKernelHighLoadIOWait`

File: blocked-edges/4.12.48-OldKernelHighLoadIOWait.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+to: 4.12.48
+from: .*
+
+url: https://issues.redhat.com/browse/COS-2705
+name: OldKernelHighLoadIOWait
+message: "After rebooting into kernel-4.18.0-372.88.1.el8_6 or later kernel nodes experience high load average and io_wait times. The nodes might fail to start or stop pods and probes may fail. Workload and host processes may become unresponsive and workload may be disrupted"
```
Comment (LalatenduMohanty) [inline blocked-edges/4.12.48-OldKernelHighLoadIOWait.yaml:6]: Lets specify the kernel with arch i.e.  4.18.0-372.89.1.el8_6.x86_64

**Review by LalatenduMohanty — DISMISSED** [review]

File: blocked-edges/4.12.48-OldKernelHighLoadIOWait.yaml (hunk 4 above)
Comment (wking) [reply blocked-edges/4.12.48-OldKernelHighLoadIOWait.yaml:6]: I'd rather not pin to an arch without someone explaining why they think this is x86_64-specific.  [OCPBUGS-30096](https://issues.redhat.com/browse/OCPBUGS-30096) mentions x86_64, but that means "at least x86_64 is exposed" and not  necessarly "only x86_64" is exposed.

Comment (LalatenduMohanty) [reply blocked-edges/4.12.48-OldKernelHighLoadIOWait.yaml:6]: I agree with Trevor. Ignore this suggestion.

**Comment by wking** [comment]
/retitle COS-2705: block edges for versions with 4.18.0-372.88.1.el8_6 or later kernel

**Review by wking — APPROVED** [review]
/lgtm

**Review by sdodson — CHANGES_REQUESTED** [review]

**Comment by wking** [comment]
4.12.51 should be signed now that #4869 has been opened.  We'll still want to [drop 4.12.48][1], but retesting just to confirm the signature presence:

/retest

[1]: https://github.com/openshift/cincinnati-graph-data/pull/4868#discussion_r1508316309

File: blocked-edges/4.11.58-OldKernelHighLoadIOWait.yaml (hunk 3 above)
Comment (wking) [reply blocked-edges/4.11.58-OldKernelHighLoadIOWait.yaml]: 83f52a86e8818d6524261483817dfe64547d5a91 uses `RHELKernelHighLoadIOWait`

File: blocked-edges/4.11.58-OldKernelHighLoadIOWait.yaml (hunk 1 above)
Comment (wking) [reply blocked-edges/4.11.58-OldKernelHighLoadIOWait.yaml]: 83f52a86e8818d6524261483817dfe64547d5a91 does not have this blank line.

File: blocked-edges/4.11.58-OldKernelHighLoadIOWait.yaml (hunk 2 above)
Comment (wking) [reply blocked-edges/4.11.58-OldKernelHighLoadIOWait.yaml]: 83f52a86e8818d6524261483817dfe64547d5a91 has the comma and period.

**Review by sdodson — APPROVED** [review]

**Review by PratikMahajan — COMMENTED** [review]
/lgtm

File: blocked-edges/4.12.48-RHELKernelHighLoadIOWait.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+to: 4.12.48
```
[the remark this answers is not in the document]
Comment (dijarvrella) [reply blocked-edges/4.12.48-RHELKernelHighLoadIOWait.yaml]: I have a version of kernel 4.18.0-372.43.1.el8_6.x86_64 which is affected by the same issue:

when trying to upgrade from 4.12.4 to 4.12.48:

`kube-controller-manager                    4.12.48   True        False         True       386d    StaticPodsDegraded: pod/kube-controller-manager-master1.ocp.example.com container "kube-controller-manager" is waiting: CrashLoopBackOff: back-off 5m0s restarting failed container=kube-controller-manager pod=kube-controller-manager-master1.ocp.example.com_openshift-kube-controller-manager(bd736b0c28527f94c70c26cf460fa064)...`

Now due to this the entire cluster upgrade has been failing

[the remark this answers is not in the document]
Comment (sdodson) [reply blocked-edges/4.12.48-RHELKernelHighLoadIOWait.yaml]: @dijarvrella Please engage with Red Hat support so that they can diagnose your issue.

**Pull request merged**